cloud/organization/organization.go

```diff
@@ -17,6 +19,10 @@ import (
 	"github.com/astronomer/astro-cli/pkg/printutil"
 )
 
+const (
+	AstronomerConnectionErrMsg = "cannot connect to Astronomer. Try to log in with astro login or check your internet connection and user permissions. If you are using an API Key or Token make sure your context is correct.\n\nDetails"
```

Comment:
small nit
```suggestion
	AstronomerConnectionErrMsg = "Cannot connect to Astronomer. Try to log in with `astro login` or check your internet connection and user permissions. If you are using an API Key or Token make sure your context is correct.\n\nDetails"
```

Reply:
The code linter won't let you capitalize the first word of an error message